hash/vsmileb_cart.xml

```diff
@@ -158,7 +158,21 @@ VTech V.Smile Baby cartridges
 	</software>
 
 	<!-- "Learn and Discover Home" -->
-	<software name="learndhg" supported="no">
+	<software name="learndhnl" supported="no">
+		<description>Dierenhuis (Netherlands)</description>
+		<year>200?</year>
+		<publisher>VTech</publisher>
+		<info name="serial" value="80-099002(NL)" />
+		<part name="cart" interface="vsmile_cart">
+			<feature name="u4" value="U4" />
+			<dataarea name="rom" size="8388608">
+				<rom name="99002 - Dierenhuis (NL).bin" size="8388608" crc="77adb3aa" sha1="0c9f96effb9e113c5e5050b074620ed27013d71f" />
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- "Learn and Discover Home" -->
+	<software name="learndhg" cloneof="learndhnl" supported="no">
```

Comment:
Does VTech actually use `g` as a region code for Germany?  If not, better to use ISO standard `de`.

Reply:
This is what all the VTech V.Smile (regular, baby, etc.) softlist are using:
VTECH CODE -> MAME suffix
US -> no suffix
UK -> uk
IT -> i
NL -> nl
GE or GR -> g
FR -> f
PT -> p
DK -> dk
SE -> sw
SP -> s
NO -> no
FI -> fi
CN -> cn
KOR -> kr

I still have to open a PR to add some dumps to 'vsmile_cart.xml', do you want me to rename all sets to start using VTech region codes on every VSmile softlist?

Reply:
Nevermind, I've changed the game suffixes to use VTech country codes. Will do the same on the other softlists on separate pull requests

Reply:
I *really* dislike these made-up schemes in software lists.  If the manufacturer has a convention, use it; otherwise use ISO.  Anything else is a recipe for confusion,